### ISSUE TITLE
Zerop for Ratios is Broken

### ISSUE THREAD

**Issue by Shinmera** [drmeister/clasp#86]
Seems like a strange one on a conceptual level, but guessing from the error it shouldn't be too hard to fix.
```
Starting MPS Clasp 0.2 ... loading image... it takes a few seconds
Top level.
> (zerop 1/2)

Condition of type: SIMPLE-PROGRAM-ERROR
File(../../include/clasp/core/numbers.h) lineNumber(120): Subclass[COMMON-LISP:RATIO] must implement method[zerop] 

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[17] CORE::REP.
 File: "/opt/clasp/clasp/build/clasp/Contents/Resources/lisp/kernel/lsp/top.lsp" (Position #22041)
```

**Comment by Shinmera** [comment]
Can confirm on b05884f.
```
> (zerop 1/2)

Condition of type: SIMPLE-PROGRAM-ERROR
File(include/clasp/core/numbers.h) lineNumber(195): Subclass[COMMON-LISP:RATIO] must implement method[zerop_] 
```

**Comment by Shinmera** [comment]
Fixed on 2e97cc5
